components/Navbar.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+
+import logo from "@/public/cesium.png";
+import Link from "next/link";
+
+const navbar = ["HOME", "TEAM", "LOGS", "EVENTS", "ABOUT US"];
```

Comment:
```suggestion
const navbar: string[] = ["HOME", "TEAM", "LOGS", "EVENTS", "ABOUT US"];
```
Let's try to use TypeScript as much as possible 💪🏼